ipam/pool.go

```diff
@@ -293,7 +293,7 @@ func (as *addressSpace) requestPool(poolId string, subPoolId string, options map
 	var ap *addressPool
 	var err error
 
-	logger.Info("Requesting pool with", zap.String("poolId", poolId), zap.Any("options", options), zap.Any("v6", v6))
+	logger.Info("Requesting a new address pool")
```

Comment:
Since you are removing details about the request, is this line needed at all? I'm asking because without any info, the log line cannot be correlated to other things. It mostly indicates that the function was invoked, but you also have a log line below that accomplishes the same thing (I don't see a code path where the other line does not get called).